bees/emailbee/emailbeefactory.go

```diff
@@ -116,9 +116,13 @@ func (factory *EmailBeeFactory) Actions() []bees.ActionDescriptor {
 				},
 				{
 					Name:        "text",
-					Description: "Content of the email",
+					Description: "Content of the email using plain text",
+					Type:        "string",
+				},
+				{
+					Name:        "html",
+					Description: "Content of the email using HTML",
 					Type:        "string",
-					Mandatory:   true,
```

Comment:
Make the "text" option mandatory and the html not-mandatory. Each email should should have a plaintext version and optionally an HTML version (as per standards).